front/src/index.html

```diff
@@ -41,12 +42,17 @@
         <script src="js/user-list.js"></script>
         <script src="js/message-list.js"></script>
         <script src="js/login.js"></script>
+        <script src="js/service.js"></script>
         <script>
             /*
              * this whole component should be mostly (or entirely re-written) 
              * when scratchy service will be done, to use the api instead of 
              * dummy data. (oh and the code mostly sucks)
              */
+             // THE TIME IS NOW
```

Comment:
Hum why did you put that as comment ? 🤔

Reply:
those comments were made by me, the first one (multiline) was before this PR and the ones after are from my old code for this issue

Reply:
idk is @js-all

Reply:
why resolve those comments need to go